keyboards/satan/satan.h

```diff
@@ -138,6 +138,36 @@
 	{k40, k41, k42, XXX, XXX, k45, XXX, XXX, XXX, XXX, k4a, k4b, k4c, k4d}  \
 }
 
+/* ISO w/ split right shift key matrix layout
+   * ,-----------------------------------------------------------.
+   * | 00 |01| 02| 03| 04| 05| 06| 07| 08| 09| 0a| 0b| 0c| 0d|49 |
+   * |-----------------------------------------------------------|
+   * | 10  | 11| 12| 13| 14| 15| 16| 17| 18| 19| 1a| 1b| 1c|     |
+   * |--------------------------------------------------------   |
+   * | 20    | 21| 22| 23| 24| 25| 26| 27| 28| 29| 2a| 2b| 2c| 2d|
+   * |-----------------------------------------------------------|
+   * | 30 | 31| 32| 33| 34| 35| 36| 37| 38| 39| 3a| 3b|  3d  | 3c|
+   * |-----------------------------------------------------------|
+   * | 40 | 41 | 42 |        45             | 4a | 4b | 4c | 4d  |
+   * `-----------------------------------------------------------'
+
+   swap 3c and 3d. 3c is right of 3d in reality
+ */
+#define KEYMAP_ISO_SPLITRSHIFT_SPLITBACKSPACE( \
```

Comment:
Could you use "LAYOUT" here instead, or add a define below to map it? 

We're depreciating KEYMAP in favor of LAYOUT.

Reply:
Done!
<sub>The commits get squashed anyway, don't they? :sweat_smile:</sub>

Reply:
Awesome, and yup.  Usually. :)